packages/react/src/video-loop/video-loop.tsx

```diff
@@ -0,0 +1,121 @@
+  /** You may pass an alternative text, complimentary to the caption. Make sure it doesn't repeat too much of the caption text, if so just a caption is sufficent. Think of this just as an alt text, but for a muted video - this text will not be visible, but read by screen readers. */
+  alt?: string;
+  className?: string;
+  rounded?: boolean;
+};
+
+type VideoLoopPropsWithAlt = VideoLoopPropsCommon & {
+  alt: string;
+};
+
+type VideoLoopPropsWithCaption = VideoLoopPropsCommon & {
+  caption: string;
+};
+
+export type VideoLoopProps = VideoLoopPropsWithAlt | VideoLoopPropsWithCaption; // Either alt or caption is required
+
+export const VideoLoop = ({
+  src,
+  format,
+  caption,
+  alt,
+  className,
+  rounded,
+}: VideoLoopProps) => {
+  // We need to check if the user prefers reduced motion, so that we can prevent the video from autoplaying if so
+  const [userPrefersReducedMotion, setUserPrefersReducedMotion] = useState<
+    null | boolean
+  >(null);
+
+  useLayoutEffect(() => {
+    const { matches: userPrefersReducedMotion } = matchMedia(
+      '(prefers-reduced-motion: reduce)',
+    );
+    setUserPrefersReducedMotion(userPrefersReducedMotion);
+    // Autoplay the video if the user does not prefer reduced motion
+    setShouldPlay(!userPrefersReducedMotion);
+  }, []);
+
+  // Control the video playback state, so that the user can pause and play the video at will
+  const [shouldPlay, setShouldPlay] = useState(false);
+  const [isPlaying, setIsPlaying] = useState(false);
+  const videoRef = useRef<HTMLVideoElement | null>(null);
+
+  useEffect(() => {
+    if (!videoRef.current) return;
+
+    if (shouldPlay) {
+      // Follow google's autoplay policy: https://developers.google.com/web/updates/2017/09/autoplay-policy-changes
+      // "Don't assume a video will play, and don't show a pause button when the video is not actually playing."
+      // "You should always look at the Promise returned by the play function to see if it was rejected:"
+      // This is why we use the promise returned by the play function, and an extra state variable to determine if the video is actually playing or not
+      videoRef.current
+        .play()
+        .then(() => setIsPlaying(true))
+        .catch(() => setIsPlaying(false));
+    } else {
+      videoRef.current.pause();
+      setIsPlaying(false);
+    }
+  }, [shouldPlay]);
+
+  return (
+    <div className={className}>
+      <div className="relative aspect-video" aria-hidden>
+        {/* Don't render the video until we have determined if the user prefers reduced motion */}
+        {userPrefersReducedMotion !== null && (
+          <>
+            <video
+              ref={videoRef}
+              // cursor-pointer is not working on the button below, so we add it here for the same effect
+              className={cx('cursor-pointer', rounded && 'rounded-3xl')}
+              autoPlay={isPlaying}
+              loop={isPlaying}
+              playsInline={isPlaying}
```

Comment:
```suggestion
              loop={isPlaying}
              playsInline={isPlaying}
```

Skal ikke disse bare være `true` uansett? Er vel bare autoPlay som påvirkes av reduced motion?

Reply:
`loop` må vi vel fortsatt ta høyde for, men ja `playsInline` kan vel alltid være `true` ja.